helm/dagster/schema/schema/charts/dagster/subschema/ingress.py

```diff
@@ -46,6 +46,7 @@ class FlowerIngressConfiguration(BaseModel):
 class Ingress(BaseModel):
     enabled: bool
     apiVersion: Optional[str]
+    additionalLabels: kubernetes.Labels
```

Comment:
thoughts on this vs just `labels`? did you cross reference any other popular charts ?

Reply:
you're right bitnami charts use `commonLabels`, `labels`, and `podLabels`